src/components/Home/UserList.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, List } from '@mui/material';
+
+import { User } from '../../interfaces/user';
+import NoResultBox from './NoResultBox';
+import UserProfile from './UserProfile';
+/*
+ * TODO
+ * 1. 클릭시 유저 프로필 페이지로 넘어가기
+ */
+
+interface Props {
+  users: User[];
+}
+
+const UserList = ({ users }: Props) => {
+  const showUserProfileList = (userProfiles: User[]) => {
+    return (
+      <List
+        sx={{
+          width: '100%',
+        }}
+      >
+        {userProfiles.map(({ _id, image, fullName, username }) => {
+          let description = null;
+
+          if (username) {
+            description = JSON.parse(username);
+          }
+
+          return (
+            <UserProfile
+              key={_id}
+              path='#'
+              image={image}
+              fullName={fullName}
+              job={description && description.job}
+              year={description && description.year}
+            />
+          );
+        })}
+      </List>
+    );
```

Comment:
```suggestion
         const job = username ? JSON.parse(username) :  JSON.parse(username).job;
         const year = username ? JSON.parse(username) : JSON.parse(username).year
         
          return (
            <UserProfile
              key={_id}
              path='#'
              image={image}
              fullName={fullName}
              job
              year
            />
          );
        })}
      </List>
    );
```

let을 사용하는 것 보다 혹시 이건 어떨까요?

Reply:
이런 피드백 너무 감사합니다:)